### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by NathanBnm**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+msgid "Many color theme choices available."
+msgstr "Plusieurs choix de couleurs de thème disponibles."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:16
+msgid "Lock position on the desktop."
+msgstr "Verrouillez la position sur le bureau."
+
+#: data/com.github.torikulhabib.nino.appdata.xml.in:17
+msgid "Save the position of the widget."
+msgstr "Enregistrez la position du widget"
```
Comment (ryonakano) [inline po/fr.po]: I suppose this translation should end with period.

File: po/fr.po [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+#: src/MainWindow.vala:222 src/MiniWindow.vala:187
+msgid "Network Is Not Available"
+msgstr "Réseau indisponible"
+
+#: src/MainWindow.vala:280
+msgid "Connect to the Internet to Monitoring Network."
+msgstr "Connectez-vous à Internet pour surveiller le réseau."
+
+#: src/MainWindow.vala:288
+msgid "Network Settings"
```
Comment (ryonakano) [inline po/fr.po]: For some reason "…" lacks from the source string, maybe you deleted it by mistake?

**Review by ryonakano — COMMENTED** [review]
I took a look at your PR and left some comments, although I can't speak French. (^^;

File: po/fr.po (hunk 2 above)
Comment (ryonakano) [reply po/fr.po]: For a reference:

screenshot from 2019-03-05 12-28-17

File: po/fr.po (hunk 1 above)
Comment (NathanBnm) [reply po/fr.po]: You're right I will fix this

File: po/fr.po (hunk 2 above)
Comment (NathanBnm) [reply po/fr.po]: I don't remember I have deleted any symbol. I will fix this too! Thank you for comment.

**Comment by NathanBnm** [comment]
@ryonakano Thanks for your comments, this is fixed now :)

**Comment by torikulhabib** [comment]
thanks

**Pull request merged**